app/src/main/java/com/example/aplikacja/EditCarFragment.kt

```diff
@@ -0,0 +1,124 @@
+package com.example.aplikacja
+
+import android.app.AlertDialog
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.EditText
+import android.widget.Spinner
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import java.io.FileWriter
+
+
+/**
+ * A simple [Fragment] subclass as the second destination in the navigation.
+ */
+class EditCarFragment : Fragment() {
+    private val selectTypeSpinner = view?.findViewById<Spinner>(R.id.selectTypeSpinner)
+    private val selectBrandSpinner = view?.findViewById<Spinner>(R.id.selectBrandSpinner)
```

Comment:
dobra, rozumiem teraz że to tylko wygląd był zmieniany, ale w sumie w takim razie usuńmy z Fragmentu wszystko co niepotrzebne